rain_api_core/aws_util.py

```diff
@@ -149,52 +150,57 @@ def get_yaml_file(bucket, key):
     except ClientError as e:
         # The specified file did not exist
         log.error("Could not download yaml @ s3://{0}/{1}: {2}".format(bucket, key, e))
+        # TODO(reweeden): remove this, why is this here!!!?
         sys.exit()
```

Comment:
Why are we calling `sys.exit` in a library function? The name of this function does not at all indicate that it would do this

Reply:
A lot of this was pasted from an ancestor app and admittedly not carefully combed through.